core/fetcher.go

```diff
@@ -47,16 +47,28 @@ func NewBlockFetcher(conn *grpc.ClientConn) (*BlockFetcher, error) {
 	}, nil
 }
 
+func (f *BlockFetcher) Start(ctx context.Context) error {
+	f.ctx, f.cancel = context.WithCancel(ctx)
```

Comment:
Is this Start supposed to be used in fx? If so, the context given to it short-lived

Reply:
No Start/Stop hooks anymore